charts/healthcheck/templates/healthcheck-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+    spec:
+      serviceAccountName: "service-catalog-healthcheck"
+      containers:
+      - name: healthcheck
```

Comment:
So, this will serve with the self-signed cert currently, right?

Reply:
Right, if existing certificate & key are found (by default it looks in /var/run/service-catalog-healthcheck but this can be overridden with --cert-dir) they will be used, otherwise it will create self signed cert.